modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -616,7 +616,7 @@
 		if kind == "list:path" then
 			return table.concat(vstudio.path(cfg, value), ';')
 		end
-	
+
```

Comment:
Can you revert this whitespace change?

Reply:
Not sure if I can do that with EditorConfig intervening upon commit.

Reply:
Huh, I did not know that github supported this. Nevermind then, its doing what it is supposed to!